test/entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+NAME="$(uname -s)"
+RELEASE="$(uname -r)"
+
+cd "$(dirname "$0")/.."
+echo
+
+if ([[ "$RELEASE" = *'boot2docker' ]] || [[ "$RELEASE" = *'moby' ]]) && \
```

Comment:
What will `$RELEASE` be on native linux hosts?  It wouldn't be `boot2docker` but I'm not sure about `moby` - is that for Docker on linux?